.changeset/quick-cups-sparkle.md

```diff
@@ -0,0 +1,5 @@
+---
+"@rocket.chat/meteor": patch
+---
+
+Change `/admin/workspace` path to `/admin/info` path as it was before 
```

Comment:
```suggestion
Rolled back a change of route from `/admin/workspace` to `/admin/info`
```

Suggestion so the message aligns with the guidelines for changesets https://developer.rocket.chat/contribute-to-rocket.chat/modes-of-contribution/participate-in-rocket.chat-development/development-workflow#4.-adding-changeset-to-your-pull-request

Reply:
I don't this that Rolled back it's a good term to use, since it's a technical term

Reply:
That's a good point, please feel free to suggest something different